Client/sdk/game/CFx.h

```diff
@@ -34,4 +36,5 @@ class CFx
     virtual void TriggerWaterSplash(CVector& vecPosition) = 0;
     virtual void TriggerBulletSplash(CVector& vecPosition) = 0;
     virtual void TriggerFootSplash(CVector& vecPosition) = 0;
+    virtual void AddParticle(eFxParticleSystems eFxParticle, CVector& vecPosition, CVector& vecDirection, float fR, float fG, float fB, float fA, bool bRandomizeColors, int iCount, float fBrightness, float fSize, bool bRandomizeSizes, float fLife) = 0;
```

Comment:
`const CVector& vecPosition` and `const CVector& vecDirection`. `int iCount` -> `std::uint32_t` or `uint32_t` (see which you prefer).